records_mover/db/connect.py

```diff
@@ -102,7 +103,10 @@ def create_sqlalchemy_url(db_facts: DBFacts,
                                  password=db_facts['password'],
                                  host=db_facts['host'],
                                  port=db_facts['port'],
-                                 database=db_facts['database'])
+                                 database=db_facts['database'],
+                                 query={
+                                     "local_infile": True  # TODO: Make generic
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker